js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-features-section.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Flex, FlexItem, FlexBlock } from '@wordpress/components';
+import { Pill } from '@woocommerce/components';
+import GridiconCheckmark from 'gridicons/dist/checkmark';
+import GridiconGift from 'gridicons/dist/gift';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import './paid-ads-features-section.scss';
+
+function FeatureList() {
+	const featuresItems = [
+		{
+			Icon: GridiconCheckmark,
+			content: __(
+				'Promote your products across Google Search, YouTube, Display, Discover, Maps, Gmail, and more.',
+				'google-listings-and-ads'
+			),
+		},
+		{
+			Icon: GridiconCheckmark,
+			content: __(
+				'Set a daily budget, and only pay when someone clicks.',
+				'google-listings-and-ads'
+			),
+		},
+		{
+			Icon: GridiconGift,
+			content: __(
+				'Claim $500 in ads credit when you spend your first $500 with Google Ads. Terms and conditions apply.',
+				'google-listings-and-ads'
+			),
```

Comment:
💅  Possible to divide the components in multiple files? 1 per component?

Reply:
Thanks for the suggestion. However, I lean toward keeping the component here because it is not used by others and it's a plain component.